README.md

```diff
@@ -1,78 +1,72 @@
+[See the official docs](https://Redotengine.org)
 for compilation instructions for every supported platform.
 
 ## Community and contributing
 
-Godot is not only an engine but an ever-growing community of users and engine
-developers. The main community channels are listed [on the homepage](https://godotengine.org/community).
+Redot is not only an engine but an ever-growing community of users and engine
+developers. The main community channels are listed [on the homepage](https://Redotengine.org/community).
 
 The best way to get in touch with the core engine developers is to join the
-[Godot Contributors Chat](https://chat.godotengine.org).
+[Redot Contributors Chat]([https://chat.Redotengine.org](https://Redotengine.org)).
 
 To get started contributing to the project, see the [contributing guide](CONTRIBUTING.md).
 This document also includes guidelines for reporting bugs.
 
 ## Documentation and demos
 
-The official documentation is hosted on [Read the Docs](https://docs.godotengine.org).
-It is maintained by the Godot community in its own [GitHub repository](https://github.com/godotengine/godot-docs).
+The official documentation is hosted on [Read the Docs]([https://docs.Redotengine.org](https://Redotengine.org)).
+It is maintained by the Redot community in its own [GitHub repository](https://github.com/Redotengine/Redot-docs).
 
-The [class reference](https://docs.godotengine.org/en/latest/classes/)
-is also accessible from the Godot editor.
+The [class reference](https://docs.Redotengine.org/en/latest/classes/)
+is also accessible from the Redot editor.
 
-We also maintain official demos in their own [GitHub repository](https://github.com/godotengine/godot-demo-projects)
-as well as a list of [awesome Godot community resources](https://github.com/godotengine/awesome-godot).
+We also maintain official demos in their own [GitHub repository](https://github.com/Redotengine/Redot-demo-projects)
+as well as a list of [awesome Redot community resources](https://github.com/Redotengine/awesome-Redot).
 
 There are also a number of other
-[learning resources](https://docs.godotengine.org/en/latest/community/tutorials.html)
+[learning resources](https://Redotengine.org)
 provided by the community, such as text and video tutorials, demos, etc.
-Consult the [community channels](https://godotengine.org/community)
+Consult the [community channels](https://Redotengine.org/community)
 for more information.
 
-[![Code Triagers Badge](https://www.codetriage.com/godotengine/godot/badges/users.svg)](https://www.codetriage.com/godotengine/godot)
-[![Translate on Weblate](https://hosted.weblate.org/widgets/godot-engine/-/godot/svg-badge.svg)](https://hosted.weblate.org/engage/godot-engine/?utm_source=widget)
-[![TODOs](https://badgen.net/https/api.tickgit.com/badgen/github.com/godotengine/godot)](https://www.tickgit.com/browse?repo=github.com/godotengine/godot)
+[![Code Triagers Badge]()]()
+[![Translate on Weblate]()]()
+[![TODOs]()]()
```

Comment:
These are empty. we should remove them unless they point to something valid.

Reply:
:+1: